firmware/src/JaamFirmware.cpp

```
@@ -1399,6 +1407,28 @@ void showTemp() {
   displayMessage(message, getNameById(DISTRICTS, settings.getInt(HOME_DISTRICT), DISTRICTS_COUNT));
 }
 
+void showEnergy() {
+  int regionId = settings.getInt(HOME_DISTRICT);
+  int status = id_to_energy[settings.getInt(HOME_DISTRICT)].first;
+  char title[38];
+  char message[35];
+  sprintf(message, "%02d%c", id_to_energy[regionId]);
```

Cppcheck (2.10-2)

[error] sprintf format string requires 2 parameters but only 1 is given.

(wrongPrintfScanfArgNum)